dali/base/dadfs.cpp

```diff
@@ -4918,13 +4910,20 @@ class CDistributedSuperFile: public CDistributedFileBase<IDistributedSuperFile>
                     if (subfroot)
                     {
                         if (!subfroot->removeProp(oquery.str()))
-                            exceptions->append(*MakeStringException(-1, "CDelayedDelete::removeEntry: SubFile %s of %s not found for removal",name?name:"(NULL)", logicalName.get()));
+                        {
+                            VStringBuffer s("SubFile %s is not owned by SuperFile %s", name, logicalName.get());
+                            if (ctx)
+                                ctx->addWuException(s.str(), 0, SeverityWarning, "DFS[clearSuperOwner]");
+                            else
+                            {
+                                Owned<IException> e = makeStringException(-1, s.str());
+                                EXCLOG(e, "DFS[clearSuperOwner]");
+                            }
+                        }
                     }
                 }
             }
         }
-        if (exceptions->ordinality())
-            throw exceptions.getClear();
```

Comment:
Yes.
The inconsistencies here, namely expecting a SuperOwner and trying to delete, but it not actually being there, should not prevent the operation continuing.